src/main/scala/sparkz/core/app/Application.scala

```diff
@@ -77,10 +78,17 @@ trait Application extends SparkzLogging {
   )
 
   protected val peerDatabase = new InMemoryPeerDatabase(settings, sparkzContext)
-  protected val peerDatabaseBackupStrategy: BackupAndRestoreStrategy = new BackupAndBackupAndRestoreFromFileStrategy(
-    FileBackupStrategyConfig(networkSettings.storageBackupDelay, networkSettings.storageBackupInterval),
-    peerDatabase.storagesToBackup()
+  private val backupDir: Path = Paths.get(settings.dataDir.getPath, "peers")
+  protected val peerDatabaseBackupStrategy: BackupAndRestoreStrategy = new BackupAndRestoreFromFileStrategy(
+    FileBackupStrategyConfig(
+      backupDir,
+      networkSettings.storageBackupDelay,
+      networkSettings.storageBackupInterval
+    ),
+    peerDatabase.storagesToBackup(backupDir.toString)
```

Comment:
just a double check. What is the result view of `backupDir.toString`  method? absolute platform specific path to the file?

Reply:
Yes, it's the platform-dependent path representation